sechub-wrapper-owasp-zap/src/main/java/com/mercedesbenz/sechub/owaspzapwrapper/config/SecHubScanConfigProvider.java

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.owaspzapwrapper.config;
+
+import java.io.File;
+import java.io.IOException;
+
+import com.mercedesbenz.sechub.commons.TextFileReader;
+import com.mercedesbenz.sechub.commons.model.SecHubScanConfiguration;
+import com.mercedesbenz.sechub.owaspzapwrapper.cli.MustExitCode;
+import com.mercedesbenz.sechub.owaspzapwrapper.cli.MustExitRuntimeException;
+
+public class SecHubScanConfigProvider {
+    public SecHubScanConfiguration getSecHubWebConfiguration(File secHubConfigFile) {
+        TextFileReader fileReader = new TextFileReader();
+
+        if (secHubConfigFile == null) {
+            return new SecHubScanConfiguration();
+        }
+        String sechubConfigJson;
+        SecHubScanConfiguration sechubScanConfig;
+        try {
+            sechubConfigJson = fileReader.loadTextFile(secHubConfigFile);
+            sechubScanConfig = SecHubScanConfiguration.createFromJSON(sechubConfigJson);
+        } catch (IOException e) {
+            throw new MustExitRuntimeException("Was not able to read sechub config file: " + secHubConfigFile, e, MustExitCode.SECHUB_CONFIGURATION_INVALID);
```

Comment:
The names of `MustExitRuntimeException` and `MustExitCode` are strange. I would expect something like `ZapWrapperRuntimeException` and `ZapWrapperExitCode`. Simply, because they make it clear, that both are your own invention.